mozfun/event_analysis/create_events_view/README.md

```diff
@@ -0,0 +1,181 @@
+### Usage
+
+```
+create_events_view(
+    view_name STRING,
+    project STRING,
+    dataset STRING,
+    funnels ARRAY<STRUCT<
+        funnel_name STRING,
+        funnel ARRAY<STRUCT<
+            step_name STRING,
+            events ARRAY<STRUCT<
+                category STRING,
+                event_name STRING>>>>>>,
+    counts ARRAY<STRUCT<
+        count_name STRING,
+        events ARRAY<STRUCT<
+            category STRING,
+            event_name STRING>>>>
+  )
+```
+
```

Comment:
Maybe this would make more sense as a bulleted list? Also, I think the code formatting (backticks) makes a bit more sense than emphasize + italics.

e.g.

```md
Parameters:

* `view_name`: ...
* `project`: ...
```